bob/playbook/playbook.go

```diff
@@ -392,8 +412,11 @@ func (p *Playbook) TaskCompleted(taskname string) (err error) {
 	errz.Fatal(err)
 
 	// TODO: use target hash?
-	err = p.pack(taskname, hashIn)
-	errz.Fatal(err)
+	// pack artifacts if only playbook disableArtifacts are set to default value (false)
+	if !p.disableArtifacts {
```

Comment:
Things like this don't read nicely (double negation). The "--no-cache" option is very common but for internal variables avoid using negative variable names.
For ref: http://davej.io/2018/05/negative-naming.html

Reply:
renamed all the variable names with negation. ex: disableCache -> enableCaching, disableArtifacts -> enableArtifacts and also changed the logic accordingly.